deploy/FlapperInit.sol

```diff
@@ -97,45 +92,35 @@ library FlapperInit {
-        FlapperUniV2Like flapper = FlapperUniV2Like(flapperInstance.flapper);
-        FlapperMomLike   mom     = FlapperMomLike(flapperInstance.mom);
+        FlapperUniV2Like flapper_ = FlapperUniV2Like(flapper);
 
         // Sanity checks
-        require(flapper.vat()      == address(dss.vat),                           "Flapper vat mismatch");
-        require(flapper.daiJoin()  == cfg.daiJoin,                                "Flapper daiJoin mismatch");
-        require(flapper.spotter()  == address(dss.spotter),                       "Flapper spotter mismatch");
-        require(flapper.pair()     == cfg.pair,                                   "Flapper pair mismatch");
-        require(flapper.receiver() == dss.chainlog.getAddress("MCD_PAUSE_PROXY"), "Flapper receiver mismatch");
+        require(flapper_.vat()      == address(dss.vat),                           "Flapper vat mismatch");
+        require(flapper_.daiJoin()  == cfg.daiJoin,                                "Flapper daiJoin mismatch");
+        require(flapper_.spotter()  == address(dss.spotter),                       "Flapper spotter mismatch");
+        require(flapper_.pair()     == cfg.pair,                                   "Flapper pair mismatch");
+        require(flapper_.receiver() == dss.chainlog.getAddress("MCD_PAUSE_PROXY"), "Flapper receiver mismatch");
 
-        PairLike pair = PairLike(flapper.pair());
+        PairLike pair = PairLike(flapper_.pair());
         address  dai  = DaiJoinLike(cfg.daiJoin).dai();
         (address pairDai, address pairGem) = pair.token0() == dai ? (pair.token0(), pair.token1())
                                                                   : (pair.token1(), pair.token0());
-        require(pairDai == dai,           "Dai mismatch");
-        require(pairGem == flapper.gem(), "Gem mismatch");
+        require(pairDai == dai,            "Dai mismatch");
+        require(pairGem == flapper_.gem(), "Gem mismatch");
 
-        require(cfg.hop == 0 || cfg.hop >= 5 minutes, "hop too low");
         require(cfg.want >= WAD * 90 / 100, "want too low");
-        require(cfg.hump > 0, "hump too low");
 
-        flapper.file("hop",  cfg.hop);
-        flapper.file("want", cfg.want);
-        flapper.file("pip",  cfg.pip);
-        flapper.rely(address(mom));
-        flapper.rely(address(cfg.caller));
+        flapper_.file("want", cfg.want);
+        flapper_.file("pip",  cfg.pip);
+        flapper_.rely(cfg.splitter);
 
-        FlapperCaller(cfg.caller).file("flapper", address(flapper));
+        SplitterLike(cfg.splitter).file("flapper", flapper);
 
-        dss.vow.file("hump", cfg.hump);
-        dss.vow.file("bump", cfg.bump);
-
-        mom.setAuthority(dss.chainlog.getAddress("MCD_ADM"));
-
-        dss.chainlog.setAddress(cfg.chainlogKey, address(flapper));
-        dss.chainlog.setAddress("FLAPPER_MOM", address(mom));
+        if (cfg.prevChainlogKey != bytes32(0)) dss.chainlog.setAddress(cfg.prevChainlogKey, address(0));
```

Comment:
need to use [removeAddress](https://github.com/makerdao/dss-chain-log/blob/master/src/ChainLog.sol#L104C14-L104C27)

Reply:
Oh right, done.